upgrading/automated_upgrades.adoc

```diff
@@ -188,7 +188,6 @@ otherwise it is left as is. For either scenario, masters are automatically
 labeled with the `master` node role, and non-master, non-infrastructure nodes
 are labeled with the `compute` node role.
 +
-ifdef::openshift-enterprise[]
```

Comment:
@ahardin-rh - this removal of ifdef wasn't followed up with an endif removal. Consequently, there is an extra endif in this file. Can you check and fix?

Reply:
@vikram-redhat Yes, fixed in https://github.com/openshift/openshift-docs/pull/9119